site/schemas/create-disruption.schema.ts

```diff
@@ -169,19 +169,30 @@ export const createDisruptionsSchemaRefined = createDisruptionSchema
             let valid = true;
             for (let i = 0; i < combinedValidity.length; i++) {
                 for (let j = i + 1; j < combinedValidity.length; j++) {
-                    const endDate = combinedValidity[i].disruptionEndDate;
-                    const endTime = combinedValidity[i].disruptionEndTime;
-
-                    if (endDate && endTime) {
-                        if (
+                    if (
+                        /* Equivalent to checkOverlap(firstStartDate: dayjs.Dayjs,
+                         *                             firstEndDate: dayjs.Dayjs,
+                         *                             secondStartDate: dayjs.Dayjs,
+                         *                             secondEndDate: dayjs.Dayjs)
+                         *  The ternary operator is to accommodate when "No end date/time" is selected for a validity */
```

Comment:
do we need this comment?

Reply:
Added the comment as the invocation of the function is quite long and might take a second to understand what is being done.